### DISCUSSION TITLE
Debutan : jarrive pas a push

### DISCUSSION THREAD

**Discussion by BlueSoley23** [community/community#62075] [Programming Help]
### Select Topic Area

Question

### Body

`` sudo git push origin main
Username for 'https://github.com': bluesoley23
Password for 'https://bluesoley23@github.com': 
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Échec d'authentification pour 'https://github.com/BlueSoley23/AzerType.git/'

Aidez moi s'il vous plait je ne m'en sors pas...
jai bien mis mon jetton comme mote de passe....

**Comment by AbdrrahimDahmani** [comment]
You need to add a PAT (Personal Access Token), GitHub is not accepting account passwords when authenticating Git operations.
refer to this Stack overflow question so you could understand how to solve it: https://stackoverflow.com/questions/68775869/message-support-for-password-authentication-was-removed
